retriever/lib/repository.py

```diff
@@ -9,7 +9,7 @@
 import imp
 from tqdm import tqdm
 from pkg_resources import parse_version
-from retriever.lib.defaults import REPOSITORY, SCRIPT_WRITE_PATH, HOME_DIR
+from retriever.lib.defaults import REPOSITORY, RETRIEVER_REPOSITORY, SCRIPT_WRITE_PATH, HOME_DIR
```

Comment:
We can use this style in places where the imports are very long. look for those places in the files you changes and clean them up 
```python
from retriever.lib.defaults import (
    REPOSITORY,
    RETRIEVER_REPOSITORY,
    SCRIPT_WRITE_PATH,
    HOME_DIR,
)
```